test/brsTypes/components/RoSGNode.test.js

```diff
@@ -2436,6 +2437,83 @@ describe("RoSGNode", () => {
+                    let childNode = new ChildComponent();
+                    let issubtype = childNode.getMethod("issubtype");
+
+                    expect(issubtype.call(interpreter, new BrsString("ChildComponent")).value).toBe(
+                        true
+                    );
+                    expect(
+                        issubtype.call(interpreter, new BrsString("ParentComponent")).value
+                    ).toBe(true);
+                    expect(issubtype.call(interpreter, new BrsString("Node")).value).toBe(true);
+
+                    let parentNode = new ParentComponent();
+                    issubtype = parentNode.getMethod("issubtype");
+
+                    expect(issubtype.call(interpreter, new BrsString("ChildComponent")).value).toBe(
+                        false
+                    );
+                    expect(issubtype.call(interpreter, new BrsString("MarkupGrid")).value).toBe(
+                        false
+                    );
+                });
+
+                it("is case-insensitive", () => {
+                    let childNode = new ChildComponent();
+                    let issubtype = childNode.getMethod("issubtype");
+
+                    expect(issubtype.call(interpreter, new BrsString("Node")).value).toBe(true);
+                    expect(issubtype.call(interpreter, new BrsString("node")).value).toBe(true);
+                    expect(issubtype.call(interpreter, new BrsString("NODE")).value).toBe(true);
+                });
+
+                it("does isSubType for other built in components", () => {
+                    let markupNode = new MarkupGrid();
+                    let issubtype = markupNode.getMethod("issubtype");
+
+                    expect(issubtype.call(interpreter, new BrsString("MarkupGrid")).value).toBe(
+                        true
+                    );
+                    expect(issubtype.call(interpreter, new BrsString("ArrayGrid")).value).toBe(
+                        true
+                    );
+                    expect(issubtype.call(interpreter, new BrsString("Node")).value).toBe(true);
+                });
+            });
+
+            describe("parentsubtype", () => {
+                it("returns the parent subtype", () => {
+                    let markupNode = new MarkupGrid();
+                    let parentsubtype = markupNode.getMethod("parentsubtype");
+
+                    let result = parentsubtype.call(interpreter, new BrsString("MarkUpGrid"));
+                    expect(result.value).toBe("ArrayGrid");
+                });
+
+                it("returns invalid if it does not exist", () => {
+                    let node = new RoSGNode([]);
+                    let parentsubtype = node.getMethod("parentsubtype");
+
+                    let result = parentsubtype.call(interpreter, new BrsString("UnknownNode"));
+                    expect(result.value).toBe(undefined);
```

Comment:
Rather than comparing `result.value` to `undefined`, consider comparing `result` directly to `BrsInvalid.Instance`.  It's a global singleton value so `.toBe` will work just fine, and will save us from having to change this test if `BrsInvalid` ever needs to store a different value.

```suggestion
                    expect(result).toBe(BrsInvalid.Instance);
```